apps/req.c

```diff
@@ -887,8 +887,6 @@ int req_main(int argc, char **argv)
                            keyout != NULL && outfile != NULL &&
                            strcmp(keyout, outfile) == 0 ? 'a' : 'w',
```

Comment:
oh, this wont work in windows...

Reply:
well, I didn't touch that code.  I only removed the "if" check after that.

Reply:
never mind.  maybe better leave some things for later...